libs/langchain/langchain/document_loaders/tensorflow_datasets.py

```diff
@@ -0,0 +1,69 @@
+from typing import Callable, Dict, Iterator, List, Optional
+
+from langchain.document_loaders.base import BaseLoader
+from langchain.schema import Document
+from langchain.utilities.tensorflow_datasets import TensorflowDatasets
+
+
+class TensorflowDatasetLoader(BaseLoader):
+    """Loads from TensorFlow Datasets into a list of Documents.
+
+    Args:
+        dataset_name: the name of the dataset to load
```

Comment:
nit: If i'm not mistaken, this documents class public attributes, but not the __init__ parameters (which do not have to be the same).

https://google.github.io/styleguide/pyguide.html#384-classes